sketch.js

```diff
@@ -48,27 +49,40 @@ var albumWordCount = function(anAlbum, aWord) {
   return wordCount;
 }
 
-// Example accessing songs in first album:
-// console.log(discography[0].name);
-// console.log('  mentions of "you": ', albumWordCount(discography[0].songs, "we"));
-// console.log('  mentions of "me": ', albumWordCount(discography[0].songs, "me"));
-
-// Another sample to see this in every album
-var countsInAllAlbums = function(aDiscography) {
-  for (var i = 0; i < aDiscography.length; i++) {
-    console.log(aDiscography[i].name)
-    console.log('  mentions of "we": ', albumWordCount(aDiscography[i].songs, "we"));
-    console.log('  mentions of "me": ', albumWordCount(aDiscography[i].songs, "me"));
+// returns an object of objects with counts
```

Comment:
Would you mind walking me through line 52 through 79? I want to be clear on why this syntax and how this code works.

@FranDias